### ISSUE TITLE
concurrent_schema_changes_test changes_while_node_down_test fails with: Column idx_token in view ks_ns2.index2_ns2_index was not found in the base table ks_ns2.cf_ns2

### ISSUE THREAD

**Issue by bhalevy** [scylladb/scylla#7647]
Seen since https://jenkins.scylladb.com/view/master/job/scylla-master/job/dtest-release/646/testReport/concurrent_schema_changes_test/TestConcurrentSchemaChanges/changes_while_node_down_test/
Scylla version 6be9f493809b8cb237751fc46ef978daa8058998
```
Traceback (most recent call last):
  File "/usr/lib64/python3.7/unittest/case.py", line 60, in testPartExecutor
    yield
  File "/usr/lib64/python3.7/unittest/case.py", line 648, in run
    self.tearDown()
  File "/jenkins/workspace/scylla-master/dtest-release/scylla-dtest/dtest.py", line 1118, in tearDown
    self.cleanUpCluster()
  File "/jenkins/workspace/scylla-master/dtest-release/scylla-dtest/dtest.py", line 1167, in cleanUpCluster
    raise AssertionError('Unexpected errors found: {}'.format(found_errors))
AssertionError: Unexpected errors found: [('node1', ['ERROR 2020-11-18 23:58:06,112 [shard 0] view - Column idx_token in view ks_ns2.index2_ns2_index was not found in the base table ks_ns2.cf_ns2', 'ERROR 2020-11-18 23:58:06,110 [shard 0] view - Column idx_token in view ks_ns2.index2_ns2_index was not found in the base table ks_ns2.cf_ns2', 'ERROR 2020-11-18 23:58:06,114 [shard 1] view - Column idx_token in view ks_ns2.index2_ns2_index was not found in the base table ks_ns2.cf_ns2', 'ERROR 2020-11-18 23:58:06,117 [shard 1] view - Column idx_token in view ks_ns2.index2_ns2_index was not found in the base table ks_ns2.cf_ns2'])]
```

**Comment by bhalevy** [comment]
@psarna I wonder if this message should be demoted to a warning like in #7597 (d43ac783c6be2001eb214d39ec54fa74662889c3)

**Comment by bhalevy** [comment]
Closing as duplicate of https://github.com/scylladb/scylla-dtest/issues/1741

**Comment by psarna** [comment]
This message is already demoted to warning, but 6be9f49 is not new enough - it was merged a couple of commits later.

**Comment by bhalevy** [comment]
> This message is already demoted to warning, but [6be9f49](https://github.com/scylladb/scylla/commit/6be9f493809b8cb237751fc46ef978daa8058998) is not new enough - it was merged a couple of commits later.

@psarna but the test keep failing consistently after d43ac78 was merged till today.
Since the message mentioned here is still printed as error rather than a warning.
If we want to keep the severity this way then we should consider ignoring the message in the dtest, unless I'm missing something.
This is the reason I closed this issue and marked as duplicate of scylladb/scylla-dtest#1741.
